beater/middleware/rate_limit_middleware.go

```diff
@@ -31,7 +31,7 @@ func SetRateLimitMiddleware(cfg *config.EventRate) Middleware {
 
 	return func(h request.Handler) (request.Handler, error) {
 		return func(c *request.Context) {
-			c.RateLimiter = store
+			c.RateLimiter = store.ForIP(c.Request)
```

Comment:
Can you change the name to include `IP` now.

Reply:
ha! i was about to do that, but then i thought you will reply with something like this: https://github.com/elastic/apm-server/pull/2706#discussion_r324647299

not sure what is the criteria now, but will change as I agree

Reply:
I don't follow. The logic that the rate limiter limits by IP is now _in the middleware_. It was not before. So now it makes sense to reflect it in the middleware name.

Reply:
As clarified offline, this is a comment on L29, not L34. 
Changed in 07888478290b0d9154e92a80c6e823b2b1b002b1